winui/Rating/Customization.md

```diff
@@ -9,4 +9,242 @@ documentation: ug
 
 # Customization in WinUI Rating
 
-This section explains the customization features available in the WinUI Rating control.
+This section explains the customization features available in the WinUI `Rating` control.
```

Comment:
@Priyanka-Vijayasankar Learn here all about the customization features such as style, item size, read-only, and template in Syncfusion WinUI Rating (SfRating) control.

Reply:
Changed the content as mentioned above.